tree/treeplayer/inc/ROOT/TTreeReaderFast.hxx

```diff
@@ -54,6 +53,13 @@ public:
       bool IsValid() const { return fEntry >= 0; }
 
    public:
+      using iterator_category = std::input_iterator_tag;
+      using value_type = const Long64_t;
+      using difference_type = Long64_t;
```

Comment:
Isn't that:
```suggestion
      using difference_type = Long_t;
```
(I.e. isn't the difference the a pointer address difference)?

Reply:
Previous definition was `public std::iterator<std::input_iterator_tag, const Long64_t, Long64_t>`, so it looks like the `difference_type` was `Long64_t`. It looks reasonable to me to keep it `Long64_t`.

Reply:
It is a bug and we should correct it in this PR.